hub/saml-engine/src/main/java/uk/gov/ida/hub/samlengine/validation/country/EidasAttributeStatementAssertionValidator.java

```diff
@@ -106,4 +110,21 @@ private void validateAttributes(Assertion assertion) {
             }
         }
     }
+
+    private QName getAttributeValueSchemaType(Response response, Attribute attribute) {
+        final QName attributeValueSchemaType = attribute.getAttributeValues().get(0).getSchemaType();
+        if (attributeValueSchemaType.getNamespaceURI().isEmpty()) {
+            Optional<Namespace> matchingPrefix = response.getNamespaces()
+                    .stream()
+                    .filter(ns -> Objects.equals(ns.getNamespacePrefix(), attributeValueSchemaType.getPrefix()))
```

Comment:
Why `Objects.equals` here? I'm suggesting this is the wrong method to use for comparison here, just curious why you've chosen it

Reply:
saves an extra NPE sanity check on `ns.getNamespacePrefix()`